src/modules/Plans/views/details/tabs/Mappings/PlanMappingsSection.tsx

```diff
@@ -161,6 +160,7 @@ export const PlanMappingsSection: FC<PlanMappingsSectionProps> = ({
         return state;
     }
   };
+  const [state, dispatch] = useReducer(reducer, initialState);
```

Comment:
??

Reply:
the mapping tab was crashing since reducer was used before initializing, refactoring is in follow up